gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/task/AttachmentMessageHandler.java

```diff
@@ -0,0 +1,84 @@
+package uk.nhs.adaptors.pss.translator.task;
```

Comment:
Rename this file, it's not an AttachementMessageHandler, it's a COPC message handler.